### PR TITLE
Kafka offset tcollector

### PR DESCRIPTION
**Pull request by krishnan-chandra**
@optimizely/data-services @jasonxh 

I refactored the Kafka offset collector that was there before and turned it into a long-running collector that picks up offsets continuously. This works but takes a long time to run (~90 seconds) so there's definitely room for improvement. Suggestions welcome.

### CODE REVIEW

File: collectors/0/secor_offset_collector.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+from collectors.lib.kafka_offset_collector import KafkaOffsetCollector
+
+
+class SecorOffsetCollector(KafkaOffsetCollector):
```
Comment (jasonxh) [inline collectors/0/secor_offset_collector.py]: i wonder how long we will still need this

File: collectors/lib/kafka_offset_collector.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    @classmethod
+    def broker(cls, broker_id):
+        return '%s/brokers/ids/%s' % (cls.KAFKA_CHROOT, broker_id)
+
+    @classmethod
+    def topics(cls, consumer_group):
+        return '%s/consumers/%s/offsets' % (cls.KAFKA_CHROOT, consumer_group)
+
+
+class KafkaOffsetCollector(object):
```
Comment (jasonxh) [inline collectors/lib/kafka_offset_collector.py]: does this only work with zk based offsets?

File: collectors/0/secor_offset_collector.py (hunk 1 above)
Comment (krishnan-chandra) [inline collectors/0/secor_offset_collector.py]: I left it in there for backwards compatibility, does secor still run on the Shadowfax Kafka? If not, I'm happy to delete this.

File: collectors/lib/kafka_offset_collector.py (hunk 2 above)
Comment (krishnan-chandra) [inline collectors/lib/kafka_offset_collector.py]: For now, yeah - if we upgrade Kafka beyond 0.8 I would want to look to some other tool like <a href="https://github.com/linkedin/Burrow">Burrow</a> to do offset lag monitoring.

**Comment by jasonxh** [comment]
LGTM

**Pull request merged**